engines/stark/ui/world/actionmenu.cpp

```diff
@@ -96,6 +99,21 @@ void ActionMenu::close() {
 	_item = nullptr;
 }
 
+Common::Rect ActionMenu::getPosition(const Common::Point &pos) {
+  Common::Rect screenWindow = getScaledPosition();
+  Common::Rect position = Common::Rect::center(pos.x, pos.y, 160, 111);
+
+  // offset screen by top border for correct boundary calculations
+  screenWindow.translate(0, _gfx->scaleHeightOriginalToCurrent(Gfx::Driver::kTopBorderHeight));
```

Comment:
The result from `getScaledPosition` should already account for the window position.